samples/gateway/Gateway_3/WebClient.Core/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Are we supposed to commit this file into the repo?

Reply:
I think we are, yes.

Reply:
Ok, to make it work when you hit F5?